src/test/java/com/ontotext/refine/cli/ExpectSystemExit.java

```diff
@@ -74,8 +74,12 @@ public void beforeEach(final ExtensionContext context) {
     }
 
     private Optional<ExpectSystemExit> getAnnotation(ExtensionContext context) {
-      return findAnnotation(context.getTestMethod(), ExpectSystemExit.class)
-          .or(() -> findAnnotation(context.getTestClass(), ExpectSystemExit.class));
+      Optional<ExpectSystemExit> opt = findAnnotation(context.getTestMethod(),
```

Comment:
I guess you've debugged something here 🤔

Reply:
No. This is simply Java 8 vs Java 11.